build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'application'
     id 'io.spring.dependency-management' version '1.0.4.RELEASE'
-    id 'org.springframework.boot' version '2.0.1.RELEASE'
+    id 'org.springframework.boot' version '2.0.2.RELEASE'
```

Comment:
Integrating with Feign caused dependency cycle which was recognised as a bug in 2.0.1 and fixed in 2.0.2.